sdk/include/psdk/winuser.h

```diff
@@ -3446,6 +3446,14 @@ typedef struct tagSOUNDSENTRYW {
 	DWORD iWindowsEffectOrdinal;
 } SOUNDSENTRYW,*LPSOUNDSENTRYW;
 
+#if(_WIN32_WINNT >= 0x0600)
+typedef struct tagAUDIODESCRIPTION {
+    UINT cbSize;   // sizeof(AudioDescriptionType)
```

Comment:
Copying from Microsoft's headers is not okay. Please delete this, type it yourself (without the comments), and match the formatting in the rest of this file.

Reply:
Ok, i thought we can use public headers and what can be found on MSDN.

Reply:
We can *learn* from publicly available information, but copying without permission is still copyright infringement.